src/sonic-config-engine/tests/test_j2files.py

```diff
@@ -26,10 +26,16 @@ def test_interfaces(self):
 
     def test_alias_map(self):
         alias_map_template = os.path.join(self.test_dir, '..', '..', '..', 'dockers', 'docker-snmp-sv2', 'alias_map.j2')
-        argument = '-m "' + self.t0_minigraph + '" -p "' + self.t0_port_config + '" -t "' + alias_map_template + '"'
+        argument = '-m ' + self.t0_minigraph + ' -p ' + self.t0_port_config + ' -t ' + alias_map_template
```

Comment:
Why remove quotes? I think it will fail when there is space in source path.

Reply:
Since all the other tests do not use quotes, I just remove them to align the format.